### ISSUE TITLE
Change volatility of prediction for different users

### ISSUE THREAD

**Issue by skurzhanskyi** [lyst/lightfm#267]
In which way I can increase the volatility of prediction for different users?
Is it may be done by change hyperparameters?

**Comment by maciejkula** [comment]
What do you mean by volatility?

**Comment by skurzhanskyi** [comment]
Now top recommends do not change or change a little, when I change user. I
want them change

**Comment by maciejkula** [comment]
What changes have you experimented with already? Try varying your hyperparameters (especially the loss functions).

**Comment by skurzhanskyi** [comment]
I use smth basic like:
```python
lfm = LightFM(loss='warp', random_state=0)
lfm.fit(mtrx_train, item_features=mtrx__feat, epochs=1000)
```
And I try to make top reccomendation for my users, but they (reccomenctions) almost do not change.
My sparsity is 0.0586%.
My mtrx_train has shape (39227, 36731).

Maybe you have any thoughts

**Comment by maciejkula** [comment]
I would recommend defining a metric that captures your goal (different recommendations for different users), then running a random search over hyperparameters to see if there are any hyperparameters that affect it. One suggestion I have is that 1000 epochs is too many, by an order of magnitude.

(This assumes that you've already checked your data and there is nothing fishy there.)

**Comment by skurzhanskyi** [comment]
Thanks for your answer. I'll try to do it

**Comment by skurzhanskyi** [comment]
Tuning hyperparameters helped. thanks!
